src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public void setSide(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getClass().getSimpleName() + ", area: "
+                + getArea() + " sq.units, color: " + getColor() + ", side: " + getSide());
```

Comment:
```suggestion
                + getArea() + " sq.units, color: " + getColor() + ", side: " + side);
```
you have direct access to fields in class